atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -0,0 +1,149 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+
+get the image manifest lists from the worker builders. If possible, group them together
+and return them. if not, return the x86_64/amd64 one.
+"""
+
+
+from __future__ import unicode_literals
+from copy import deepcopy
+import requests
+import requests.auth
+
+from six.moves.urllib.parse import urlparse
+
+from atomic_reactor.plugin import PostBuildPlugin, PluginFailedException
+from atomic_reactor.util import Dockercfg
+
+
+class GroupManifestsPlugin(PostBuildPlugin):
+    key = 'group_manifests'
+    is_allowed_to_fail = False
+
+    def __init__(self, tasker, workflow, registries, group=True, goarch=None):
+        """
+        constructor
+
+        :param tasker: DockerTasker instance
+        :param workflow: DockerBuildWorkflow instance
+        :param registries: dict, keys are docker registries, values are dicts containing
+                           per-registry parameters.
+                           Params:
+                            * "secret" optional string - path to the secret, which stores
+                              login and password for remote registry
+        :param group: bool, if true, return the grouped image manifests from the workers
```

Comment:
Nit: better description might be "if true, create a manifest list; otherwise only add tags to amd64 image manifest"